pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -197,68 +197,72 @@ set -e
 
 KEK_NAME=%s
 KEY_PATH=%s
-CURL_PAYLOAD=$(mktemp)
-ARGS=(--silent --show-error --request GET --header "X-Vault-Token: ${VAULT_TOKEN//[$'\t\r\n']}")
-PYTHON_DATA_PARSE="['data']"
+export VAULT_TOKEN=${VAULT_TOKEN//[$'\t\r\n']}
 
-# If a vault namespace is set
-if [ -n "$VAULT_NAMESPACE" ]; then
-  ARGS+=(--header "X-Vault-Namespace: ${VAULT_NAMESPACE}")
-fi
+if command -v vault; then
+  vault kv get -field="$KEK_NAME" "$VAULT_BACKEND_PATH"/"$KEK_NAME" > "$KEY_PATH"
```

Comment:
I just wondering if the tls certs are specified do we need to pass that for the `vault` binary like for the cURL command?